.github/workflows/unit-test.yml

```diff
@@ -0,0 +1,51 @@
+name: UnitTests
+
+on:
+  pull_request:
+    branches: 
+     - "**"
+
+jobs:   
+  run-unit-tests:
+
+    runs-on: ubuntu-latest
+    container: compucorp/civicrm-buildkit:1.0
+    env:
+      CIVICRM_CORE: 5.24.6
+      SHOREDITCH_TAG: 0.1.0-alpha.37
+
+    services:
+      mysql:
+        image: mysql:5.7
+        env:
+          MYSQL_ROOT_PASSWORD: root
+        ports:
+        - 3306
+        options: --health-cmd="mysqladmin ping" --health-interval=10s --health-timeout=5s --health-retries=3
+
+    steps:
+
+      - uses: actions/checkout@v2
+
+      - name: Config mysql database
+        run: echo "SET GLOBAL sql_mode=(SELECT REPLACE(@@sql_mode,'ONLY_FULL_GROUP_BY',''));" | mysql -u root --password=root --host=mysql
+
+      - name: Config amp
+        run : amp config:set --mysql_dsn=mysql://root:root@mysql:3306
+      
+      - name: Build Drupal site 
+        run: civibuild create drupal-clean --civi-ver ${CIVICRM_CORE} 
+ 
+      - name: Prepare CiviCase Extension for testing
+        run : |
+          cp -R ../uk.co.compucorp.civicase /buildkit/civicrm-buildkit/build/drupal-clean/web/sites/all/modules/civicrm/tools/extensions
+          cd /buildkit/civicrm-buildkit/build/drupal-clean/web/sites/all/modules/civicrm/tools/extensions 
+          git clone -b ${SHOREDITCH_TAG} --single-branch https://github.com/civicrm/org.civicrm.shoreditch.git 
+          drush cc all && drush cc civicrm
+          drush cvapi Extension.install key=org.civicrm.shoreditch
+          drush cvapi Extension.install key=uk.co.compucorp.civicase
```

Comment:
since the docker image used here comes with buildkit, it's much easier to use `cv` for this. These two drush commands can be replaced with `cv en shoreditch && cv en civicase`

Reply:
This has been fixed.